calico/felix/fetcd.py

```diff
@@ -257,7 +339,9 @@ def _run(self):
         self.load_config.wait()
         while True:
             _log.info("Reconnecting and loading snapshot from etcd...")
-            self._reconnect(copy_cluster_id=False)
+
+            _reconnect(self, copy_cluster_id=False)
+
```

Comment:
You seem to be quite a fan of blank lines! :) Not convinced they add anything to readability here.